client/internal/routemanager/systemops/systemops_windows.go

```diff
@@ -230,10 +230,13 @@ func (rm *RouteMonitor) parseUpdate(row *MIB_IPFORWARD_ROW2, notificationType MI
 	if idx != 0 {
 		intf, err := net.InterfaceByIndex(idx)
 		if err != nil {
-			return update, fmt.Errorf("get interface name: %w", err)
```

Comment:
Based on the description of the PR, it was to help debug things only. But if you remove the return the logic will be different. Or not?